Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "reedline"
-version = "0.6.0"
+version = "0.7.0"
 authors = ["JT <jonathan.d.turner@gmail.com>"]
```

Comment:
at some point we may want to change this to 'The Nushell Project Developers', or whatever nushell says.

Reply:
Oops too late :D

Reply:
it can wait until next time - no biggy